src/test/java/myPackage/MyStepdefs.java

```diff
@@ -10,14 +10,13 @@
 import org.openqa.selenium.Keys;
 import org.openqa.selenium.chrome.ChromeDriver;
 
-public class MyStepdefs {
-    ChromeDriver browser;
+public class MyStepdefs extends Driver {
+    Driver d = new Driver();
+    ChromeDriver browser = d.driver();
```

Comment:
why using chromdriver, why not use webdriver class ? 

am i missing something ?

Reply:
thanks making changes accordingly